advanced/index.md

```diff
@@ -94,14 +94,14 @@ $ git log -G [regex-pattern-in-patch]
 $ git log --grep=[regex-in-message]
 ```
 
-Filtering by file state, *added* (A), *modified* (M), or *deleted* (D), also narrows what change requires assessment.
+Filtering by file state, i.e. *added* (A), *modified* (M), or *deleted* (D):
```

Comment:
Good point, will re-add a phrase motivating the use of these flags.

E.g. means "for example," while i.e. stands for "that is to say." A, M, and D are the only three states available (should we add untracked?), so the reference to state implies one of those three options -- there are no other example states.